### ISSUE TITLE
[Word Desktop]: NVDA is not announcing any proofing issues except Spelling error.

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8276]
### Steps to reproduce:
1. Open the attached sample on Win32 desktop app
2. Navigate through the doc using down arrow
3. Observe that when misspelled word encounters NVDA is announcing the "Spelling error"
4. Now navigate to some grammatical error (Blue underlined).
5. Observe that No announcement by NVDA.
6. Now again navigate to some "Advanced Proofing error" (Brown colored dotted underlined)
7. Observe that there is no announcement by NVDA.

### Expected behavior:
NVDA should announce the grammatical as well as proofing error.

### Actual behavior:
 NVDA is not announcing anything while Navigating throughout the document about the grammatical as well as Advanced proofing error.

### System configuration:
NVDA version: 2018.1.1
App Name: Word

NVDA Installed or portable: Installed

**Other information:**

Windows version:
> Example: Windows 10 Version 1803 Build 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by WiproPSAccTest** [comment]
![proofing advanced](https://user-images.githubusercontent.com/22956130/40101094-f0f1b152-5903-11e8-8d20-2f4a0426c530.png)

**Comment by Adriani90** [comment]
Closing as duplicate of #5877
